docs/src/modules/components/ComponentLinkHeader.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/jsx-no-target-blank */
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import GitHubIcon from '@material-ui/icons/GitHub';
+import SketchIcon from 'docs/src/modules/components/SketchIcon';
+import FigmaIcon from 'docs/src/modules/components/FigmaIcon';
+import BundleSizeIcon from 'docs/src/modules/components/BundleSizeIcon';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import W3CIcon from 'docs/src/modules/components/W3CIcon';
+import MaterialDesignIcon from 'docs/src/modules/components/MaterialDesignIcon';
+import { useSelector } from 'react-redux';
+import { makeStyles, fade } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: 0,
+    padding: 0,
+    listStyle: 'none',
+    display: 'flex',
+    flexWrap: 'wrap',
+    marginBottom: theme.spacing(2),
+    '& li': {
+      margin: theme.spacing(0.5),
+      padding: theme.spacing(0.5, 1, 0.5, 0.5),
+      border: `1px solid ${fade(theme.palette.action.active, 0.12)}`,
+      borderRadius: theme.shape.borderRadius,
+    },
+    '& a': {
+      color: theme.palette.text.primary,
+      textDecoration: 'none',
+      display: 'flex',
+      alignItems: 'center',
+    },
+    '& svg': {
+      marginRight: theme.spacing(1),
+    },
+  },
+}));
+
+export default function ComponentLinkHeader(props) {
+  const { headers } = props;
+  const classes = useStyles();
+  const t = useSelector((state) => state.options.t);
+
+  return (
+    <ul className={classes.root}>
+      <li>
+        <a
+          target="_blank"
+          rel="noopener nofollow"
+          href={`https://github.com/mui-org/material-ui/tree/next/packages/material-ui/src/${headers.components[0]}`}
+        >
+          <GitHubIcon />
+          {t('viewGitHubSort')}
+        </a>
+      </li>
+      {headers.materialDesign ? (
+        <li>
+          <a target="_blank" rel="noopener nofollow" href={headers.materialDesign}>
```

Comment:
Let's not make these blank. I don't see any reason to open them in a new window since page state is not critical. https://www.w3.org/TR/WCAG20-TECHS/G200.html for more use cases of _blank none of which apply here as far as I can tell

Reply:
I have removed all the blank links.